src/Test/All.hs

```diff
@@ -8,6 +9,10 @@ import Data.List
 import System.Directory
 import System.FilePath
 
+#if __GLASGOW_HASKELL__ < 710
+import Data.Functor ((<$>))
+#endif
```

Comment:
The code like `__GLASGOW_HASKELL__ < 710` is self-descriptive and will be dropped when you quit supporting previous versions.

`import Data.Functor; import Prelude` looks strange and will be polluting the code after versions < 7.10 support is dropped.